atoma-proxy/src/server/streamer.rs

```diff
@@ -501,20 +473,23 @@ impl Stream for Streamer {
                         ))));
                     };
 
-                    if let Some(usage) = chunk.get(USAGE) {
```

Comment:
This seems unrelated to total hash updates. Is the logic being replaced equivalent?

Reply:
The `UpdateStackTotalHash` is what ultimately concatenates the hash in the stacks table